src/main/java/com/kumarvv/table2pojo/core/PojoWriter.java

```diff
@@ -236,7 +236,13 @@ private String generatePojo(final String tableName, final List<DbColumn> columns
         methods.forEach(s -> sb.append(s).append(NEW_LINE));
 
         sb.append("}");
-
+        writePojo(tableName+"Repository","import org.springframework.data.jpa.repository.JpaRepository;\n" + 
```

Comment:
@skaros I want to keep the POJO writer as generic as possible and not tie with any specific framework. 

And yes, we can add support for frameworks or libraries with optional CLI parameters. For an example, if we pass `-s` for the spring framework then we can inject these statements conditionally. 

Please keep this PR for the capitalization fix only. You may create a new PR or fork for spring framework support. 

Thanks for your help again,
Vijay